interfaces/CMakeLists.txt

```diff
@@ -57,7 +51,6 @@ if (WRAP_SOQT)
         OUTPUT_DIR ${CMAKE_BINARY_DIR}/pivy/gui
         SOURCES soqt.i)
 
-    link_directories(${SoQt_LIBRARY_DIRS})
 
 
     target_include_directories(soqt
```

Comment:
You may add the `INTERFACE` keyword here, e.g. `target_include_directories(soqt INTERFACE ${CMAKE_SOURCE_DIR}/interfaces)`

Reply:
INTERFACE doesn't work here, but I am not sure what INTERFACE means in this context.

Reply:
See [this thread](https://stackoverflow.com/questions/26243169/cmake-target-include-directories-meaning-of-scope) on stackoverflow for meaning of those keywords. I agree that `PUBLIC` is the right setting for this (you need the includes to compile your sources and dependent targets need those include files, too. I mixed that up, as always :).